### DISCUSSION TITLE
How do you execute short?

### DISCUSSION THREAD

**Discussion by ScuzzyHoulihan** [hootnot/oanda-api-v20#188] [Q&A] [answered]
Was able to open a trade and close. But what's the cmd to execute a short position?

Awesome project by the way!

**Comment by LPMIV** [comment, marked as answer]
Give the price with a negative symbol.

**Reply by hootnot** [reply]
the units should be negative for a short

https://developer.oanda.com/rest-live-v20/order-ep/#collapse_2_example_curl_4

**Reply by ScuzzyHoulihan** [reply]
Awesome my execution is working for shorts thanks!
